### PR TITLE
Add support for KeyCloak and database auto initialization using docke…

### PR DESCRIPTION
**Pull request by bharatak**
…r-compose file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,57 @@ all the dependencies automatically.
...
+mvn clean install
+java -Djdbc.url=jdbc:postgresql://localhost:5432/consent_manager -Djdbc.username=postgres -Djdbc.password=password -jar target/cmdb-initializer-1.0-SNAPSHOT.jar
+docker volume rm consent-manager_postgres_data
+```
+  
+3. After you Keycloak setup, you'll need to keep track of the Secret associated to consent-manager 
+* Navigate to ConsentManager Realm
+* Go to "Clients" > "Credentials"
+* Get the Secret field
+* Update the secret in [docker-compose.yml](docker-compose.yml) under variables CONSENTMANAGER_CLIENTREGISTRY_XAUTHTOKEN & CONSENTMANAGER_KEYCLOAK_CLIENTSECRET
```
Comment (ganesanarun) [inline README.md:100]: * CONSENTMANAGER_CLIENTREGISTRY_XAUTHTOKEN => this one needs to be copied from central-registry realm
* CONSENTMANAGER_KEYCLOAK_CLIENTSECRET => this one needs to be copied from ConsentManager realm

File: src/main/resources/application-docker.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+server:
+  port: 9000
+consentmanager:
+  consentservice:
+    defaultPageSize: 20
+    maxPageSize: 100
+    url: http://consent-manager:9000
+  clientregistry:
+    XAuthToken: efb60576-8110-47df-ad94-3883924624e6
```
Comment (ganesanarun) [inline src/main/resources/application-docker.yml:9]:
```suggestion
    XAuthToken: {{CONSENTMANAGER_CLIENTREGISTRY_XAUTHTOKEN}}
```

File: src/main/resources/application-docker.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    userCreationTokenValidity: 5
+    userIdSuffix: "@ncg"
+  otpservice:
+    url: http://otp:80
+    identifiers:
+      - MOBILE
+  keycloak:
+    baseUrl: http://keycloak:8080/auth
+    clientId: consent-manager
+    clientSecret: efb60576-8110-47df-ad94-3883924624e6
```
Comment (ganesanarun) [inline src/main/resources/application-docker.yml:26]:
```suggestion
    clientSecret: {{CONSENTMANAGER_KEYCLOAK_CLIENTSECRET}}
```

File: README.md (hunk 1 above)
Comment (bharatak) [reply README.md:100]: Got it.. Will fix the readme

Comment (bharatak) [reply README.md]: Done

**Pull request merged**